src/types/KindeSDK.ts

```diff
@@ -74,3 +74,14 @@ export type FeatureFlag = {
     v?: FeatureFlagValue; // v -> value
     t?: FeatureFlagType; // t -> type, s -> string, b -> boolean, i -> integer
 };
+
+export type LoginAdditionalParameters = Omit<
```

Comment:
Small nitpick. I think using `unknown` would be more appropriate here instead of `any` since this will force the end-user to cast the corresponding value to a concrete `type` before using it.
 
 ```ts
 export type LoginAdditionalParameters = Omit<
    OrgAdditionalParams,
    'is_create_org'
> & {
    [key: string]: unknown;
};
 ```